test/support/commands.js

```diff
@@ -30,6 +29,10 @@ Cypress.Commands.add('showPreview', (token, fileId, options) => {
 Cypress.Commands.add('showControls', () => {
     cy.getByTestId('bp').trigger('mouseover');
     cy.getByTestId('bp-controls').should('be.visible');
+    // Elements with an opacity of 0 are now considered not visible in 6.0+
+    // This interaction forces the controls to be visble
+    // See https://github.com/cypress-io/cypress/issues/14191#issuecomment-745730897
+    cy.getByTitle('Click to enter page number').click();
```

Comment:
We should move these back, as the controls pane is reused across all viewers, some of which don't use page controls. Mousing over the controls should set their opacity to 1, so it seems like there's an underlying issue with the steps above.